CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- changed: Include `currencyCode` in `spendInfo` to `getMaxSpendable`
```

Comment:
`- fixed: Provide a fallback currencyCode to EdgeCurrencyEngine.getMaxSpendable`